packages/shared/components/Sidebar.svelte

```diff
@@ -19,13 +19,47 @@
     let prevPartiallyUnstakedAmount = 0 // store the previous unstaked funds to avoid notifying when unstaked funds decrease
     let showStakingNotification = false
 
-    const hasTitleBar = document.body.classList.contains('platform-win32')
     const profileColor = 'blue' // TODO: each profile has a different color
 
     $: profileInitial = getInitials($activeProfile?.name, 1)
     $: healthStatus = $networkStatus.health ?? 0
     $: $dashboardRoute, $stakingEventState, $partiallyUnstakedAmount, manageUnstakedAmountNotification()
 
+    $: $dashboardRoute, $activeProfile?.hasVisitedStaking || showStakingNotification, updateSidebar()
+
+    let sidebarTabs = [
+        {
+            active: false,
+            icon: 'wallet',
+            label: locale('tabs.wallet'),
+            route: Tabs.Wallet,
+            notificationType: undefined,
+            onClick: openWallet,
+        },
+        {
+            active: false,
+            icon: 'tokens',
+            label: locale('tabs.staking'),
+            route: Tabs.Staking,
+            notificationType: undefined,
+            onClick: openStaking,
+        },
+    ]
+
+    function updateSidebar() {
+        sidebarTabs = sidebarTabs.map((tab) => {
+            tab.active = tab.route === $dashboardRoute
+            if (Tabs.Staking === tab.route) {
+                tab.notificationType = !$activeProfile?.hasVisitedStaking
+                    ? 'warning'
```

Comment:
I think we should use the NotificationType Enums here

Reply:
using global notification type now